src/Generate/Traits/Names.php

```diff
@@ -85,7 +85,7 @@ protected function initCommonNames(
 
         $controllerFullName = $controllerGenerator->qualifyClass($controllerName);
         if (!Str::startsWith($controllerFullName,
-            $startsWith = trim($controllerGenerator->rootNamespace(), '\\') . '\Http\\Controllers\\')) {
+            $startsWith = trim($controllerGenerator->rootNamespace(), '\\') . '\Http\\Controllers\\Admin\\')) {
```

Comment:
What is fixed by this? Or why we change this?

Reply:
We changed this because this function is used only in routes generation and now we used Admin group instead of full name of controller.